pandas/sparse/series.py

```diff
@@ -654,6 +656,48 @@ def combine_first(self, other):
         dense_combined = self.to_dense().combine_first(other)
         return dense_combined.to_sparse(fill_value=self.fill_value)
 
+    def to_coo(self, ilevels=(0,), jlevels=(1,), sort_labels=False):
+        """
+        Create a scipy.sparse.coo_matrix from a SparseSeries with MultiIndex.
+
+        Use ilevels and jlevels to determine the row and column coordinates respectively.
+        ilevels and jlevels are the names (labels) or numbers of the levels.
+        {ilevels, jlevels} must be a partition of the MultiIndex level names (or numbers).
+
+        Parameters
+        ----------
+        ilevels : tuple/list
+        jlevels : tuple/list
+        sort_labels : bool, default False
+            Sort the row and column labels before forming the sparse matrix.
+
+        Returns
+        -------
+        y : scipy.sparse.coo_matrix
+        il : list (row labels)
```

Comment:
Also, wouldn't it be more useful for this function to return a `MultiIndex` for the `rows`/`columns` rather than native Python lists? You can do lots of stuff to lookup values in a MultiIndex (e.g., `get_loc`/`get_indexer`/`slice_indexer`) even if you don't have a dataframe to store them in -- not so much with plain lists.